### ISSUE TITLE
Menu padding issues in gtk3 menuitems in xfce 4.12

### ISSUE THREAD

**Issue by gps08** [B00merang-Project/Chrome-OS#16]

**Comment by Elbullazul** [comment]
@gps08 screenshot comparing with another theme please?

**Comment by gps08** [comment]
with default greybird theme:
![with_greybird](https://user-images.githubusercontent.com/19977242/27262168-bd922174-546e-11e7-957e-42e4c88aa1d6.png)

with chrome-os theme
![with_chrome_os](https://user-images.githubusercontent.com/19977242/27262169-bffe9af0-546e-11e7-95d4-0187413a5d88.png)

**Comment by Elbullazul** [comment]
this is a weird issue we've been having. I haven't been able to find a solution for now. What's even weirder is the icons do appear on certain apps.

will check greybird code

**Comment by gps08** [comment]
i think its the apps that use gtk3.2 in particular

**Comment by Elbullazul** [comment]
@gps08 I think I found the bug. I'm missing min-height/width for menuitemm check/radio

can you try to add min sizes on your side and see if it works? I'm unavailable to test for the moment

**Comment by Elbullazul** [comment]
@gps08 applied and pushed fix, at least for check radio buttons

**Comment by gps08** [comment]
@Elbullazul  yes this fixes the issue, thanks
![image](https://user-images.githubusercontent.com/19977242/27316756-983fbba2-559f-11e7-9c7a-a40dd2aae6de.png)

I think the spacing can be a little better though

**Comment by Elbullazul** [comment]
reduced spacing by 3px

cheers

**Comment by gps08** [comment]
i meant horizontal spacing, but the one in the commit changes the vertical one
![image](https://user-images.githubusercontent.com/19977242/27650990-70b0faa4-5c54-11e7-8b8d-5e20873f31c7.png)

**Comment by Elbullazul** [comment]
@gps08 like I said in the commit, I got my CSS messed up a bit. I'm restoring padding to 6px 8px

pushed changes in 656f510

**Comment by Elbullazul** [comment]
it should work fine this time with 8689082

**Comment by gps08** [comment]
@Elbullazul  actually its still not okay

![image](https://user-images.githubusercontent.com/19977242/27687105-2288eeaa-5cf3-11e7-913a-c9eff7f1209f.png)

the entry with the checkbox is lot more thicker than the other ones

**Comment by Elbullazul** [comment]
@gps08 what version of Xfce are you using and how could I get it? on my arch VM I still have mousepad GTK2

**Comment by gps08** [comment]
@Elbullazul  its xubuntu 17.04, heres a [link](https://xubuntu.org/getxubuntu/)
i think it ported to gtk 3.2 just recently

**Comment by Elbullazul** [comment]
@gps08 I have to keep the padding like this, since nemo breaks if it's not set this way:

![nemo-check](https://user-images.githubusercontent.com/15310985/27718704-9384c976-5d1b-11e7-8bf4-cd475f1330e2.png)

and mousepad is still not complete, so it may have something to do.

or I could reduce radiocheck size, I'll evaluate fixes

**Comment by gps08** [comment]
u sure that nemo uses gtk3.2??

**Comment by Elbullazul** [comment]
yes, it uses the installed GTK 3 toolkit

but you see that in nemo the check is smaller than in mousepad, so maybe I can have the mousepad check size reduced

what do you think?

**Comment by gps08** [comment]
the xfce-panel wifi menu also shows the same problem as in mousepad, so i am thinking its a global problem
![image](https://user-images.githubusercontent.com/19977242/27737801-217a2b76-5dc7-11e7-8b06-8e130d10aa4f.png)

**Comment by gps08** [comment]
try moving the gtk3.2 folder and see if that breaks the nemo menuitems, i think it must be using gtk3.0 folder of the theme rather than gtk3,2

**Comment by Elbullazul** [comment]
@gps08 no, Nemo uses 3.20. You can't have 3.18 and 3.20 apps running together at the same time, since GTK MUST ONLY be installed once on a system, else there's big problems

also, I fixed the check in nemo tweaking the 3.20 gtk-widgets.css, so it's obvious it uses 3.20

**Comment by gps08** [comment]
@Elbullazul  okay, but i have 3 apps that i can pinpoint having this problem..

**Comment by Elbullazul** [comment]
@gps08 maybe  I got messed up with my words.

I was saying that Nemo has weird placing issues with the radiocheck items. I acknowledge the Xfce bugs too, I was just pointing the fact that Xfce is not the only platform affected. 

now, since the check-radio icons are 16x16, it needs bigger margins to stay aligned in nemo and other apps. So the only way of fixing this for all current and future apps is to reduce the check-radio size to, say, 10x10. I will test this and I installed Xubuntu in a VM, so I'll be able to live-test results.

I'll post as soon as I push modifications. Thanks again for your help
